transformers/test_data/vat_grab.go

```diff
@@ -97,22 +93,18 @@ var VatGrabHeaderSyncLogWithNegativeDink = core.HeaderSyncLog{
 	Transformed: false,
 }
 
-var VatGrabModelWithNegativeDink = shared.InsertionModel{
+var VatGrabModelWithNegativeDink = event.InsertionModel{
```

Comment:
I think we should be using `CopyEventModel` here and not exporting the `event.InsertionModel` directly (so that modifications to properties on the var don't pollute other tests)